ui/platform_window/x11/x11_window_manager_ozone.cc

```diff
@@ -10,11 +10,19 @@ namespace ui {
 
 X11WindowManagerOzone::X11WindowManagerOzone() : event_grabber_(None) {}
 
-X11WindowManagerOzone::~X11WindowManagerOzone() {}
+X11WindowManagerOzone::~X11WindowManagerOzone() {
+  DCHECK(x11_windows_.empty());
+}
 
 void X11WindowManagerOzone::GrabEvents(XID xwindow) {
-  if (event_grabber_ != None)
```

Comment:
this check seems to mean the following:

if another window has 'grab' property, bail out. To me it means, we need to call ungrab before to call grab with a different window.

can we call ungrab before and keep the if-check as is? this would also allow us to remove the lines below:

+  if (event_grabber_) {
￼ +    X11WindowOzone* x11_window = GetX11Window(event_grabber_);
￼ +    x11_window->OnLostCapture();
￼ +  }